docs/pjrt_integration.md

```diff
@@ -2,23 +2,37 @@
+[PJRT](https://github.com/openxla/xla/blob/main/xla/pjrt/c/pjrt_c_api.h) is a uniform Device API that simplifies the growing complexity of ML workload execution across hardware and frameworks. It provides a hardware and framework independent interface for compilers and runtimes. The long term vision is that: (1) frameworks (JAX, TF, etc.) will call PJRT, which has device-specific implementations that are opaque to the frameworks; (2) each device focuses on implementing PJRT APIs, and can be opaque to the frameworks.
 
 This doc focuses on the recommendations about how to integrate with PJRT, and how to test PJRT integration with JAX.
 
+### What is PJRT
+
+The PJRT C API provides the device compiler and runtime interface, as well as the interface between the framework and the hardware.
+
+A PJRT plugin is a Python package that contains a `.so` file which implements PJRT C APIs, as well as Python methods/setup to make it discoverable by the framework.
+
+Examples include:
+
+- [Apple Metal plugin](https://developer.apple.com/metal/jax/)
+- [CUDA plugin](https://pypi.org/project/jax-cuda12-pjrt/)
+- [Intel GPU plugin](https://intel.github.io/intel-extension-for-tensorflow/latest/docs/guide/OpenXLA_Support_on_GPU.html)
+- [TPU plugin](https://storage.googleapis.com/jax-releases/libtpu_releases.html)
+
 ## How to integrate with PJRT
 
-### Step 1: Implement [PJRT C API interface](https://github.com/openxla/xla/blob/71a4e6e6e4e9f0f8b8f25c07a32ad489aff19239/xla/pjrt/c/pjrt_c_api.h)
+### Step 1: Implement [PJRT C API interface](https://github.com/openxla/xla/tree/main/xla/pjrt/c/pjrt_c_api.h)
 
 **Option A**: You can implement the PJRT C API directly.
 
 **Option B**: If you're able to build against C++ code in the [xla repo](https://github.com/openxla/xla) (via forking or bazel), you can also implement the PJRT C++ API and use the C→C++ wrapper:
 
-1. Implement a C++ PJRT client inheriting from the [base PJRT client](https://github.com/openxla/xla/blob/main/xla/pjrt/pjrt_client.h) (and related PJRT classes). Here are some examples of C++ PJRT client: [pjrt\_stream\_executor\_client.h](https://github.com/openxla/xla/blob/c23fbd601a017be25726fd6d624b22daa6a8a4e5/xla/pjrt/pjrt_stream_executor_client.h), [tfrt\_cpu\_pjrt\_client.h](https://github.com/openxla/xla/blob/c23fbd601a017be25726fd6d624b22daa6a8a4e5/xla/pjrt/tfrt_cpu_pjrt_client.h).
-1. Implement a few C API methods that are not part of C++ PJRT client:
+1. Implement a C++ PJRT client inheriting from the [base PJRT client](https://github.com/openxla/xla/blob/main/xla/pjrt/pjrt_client.h) (and related PJRT classes). Here are some examples of C++ PJRT client: [pjrt\_stream\_executor\_client.h](https://github.com/openxla/xla/blob/main/xla/pjrt/pjrt_stream_executor_client.h), [cpu\_client.h](https://github.com/openxla/xla/blob/main/xla/pjrt/cpu/cpu_client.h).
```

Comment:
Let me know if this is the right link to replace here.

Reply:
cc @changm